be/src/runtime/decimalv2_value.cpp

```diff
@@ -192,7 +192,7 @@ DecimalV2Value operator/(const DecimalV2Value& v1, const DecimalV2Value& v2) {
     int128_t x = v1.value();
     int128_t y = v2.value();
 
-    //todo: return 0 for divide zero
+    DCHECK(y != 0);
     if (x == 0 || y == 0) return DecimalV2Value(0);
```

Comment:
```suggestion
    if (x == 0) return DecimalV2Value(0);
```

Reply:
we have to do this check to prevent divide / zero which will cause core in BE.